recipes/openfx/all/CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+cmake_minimum_required(VERSION 3.1)
```

Comment:
Who provided this cmake recipe? Is it possible to submit it to the upstream?

Reply:
I provided it, and yes it is possible.